### PR TITLE
Finished function and testing

### PR DESCRIPTION
**Pull request by Manpreet-Bhatti**
Hopefully the 1-to-1 mapping was done correctly -- added a new type explicitly for Icon.

### CODE REVIEW

File: src/tree/types.ts [hunk 1]
```diff
@@ -24,6 +24,14 @@ export enum FileType {
   CONFIG_FILE = 'CONFIG_FILE',
 }
 
+export enum Icon {
+  /* Added one space after emoji so that it doesn't stick together
+  with the next phrase or delete a character */
+  FOLDER = '📂 ',
```
Comment (ralph-dev) [inline src/tree/types.ts:30]: This should be done somewhere else, not here. This is just a Map, don’t have one function do two things.

File: src/utils/getFileIconFromFileType.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+/** 1-to-1 maps a given file type to an Icon type, returning an emoji that
+ * corresponds to the specific file type
+ * @param {FileType} fileType - a type a file such as FileType.FILE, FileType.FOLDER,
+ * or FileType.CONFIG_FILE
+ * @returns {Icon} - an emoji passed into a string depending on which file
+ * type was found (FILE, CONFIG_FILE, or FOLDER)
+ */
+
+export const getFileIconFromFileType = (fileType: FileType): Icon => {
+  if (fileType === FileType.CONFIG_FILE) {
```
Comment (ralph-dev) [inline src/utils/getFileIconFromFileType.ts:12]: Use something like Map[FileTypePassedin]
That way you don’t need an if statement for each map....

Comment (Manpreet-Bhatti) [reply src/utils/getFileIconFromFileType.ts]: Added that instead of return statement (Icon[fileType])

File: src/tree/types.ts (hunk 1 above)
Comment (Manpreet-Bhatti) [reply src/tree/types.ts]: Added this to the function file instead

File: src/utils/getFileIconFromFileType.ts [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+ * @param {FileType} fileType - a type a file such as FileType.FILE, FileType.FOLDER,
+ * or FileType.CONFIG_FILE
+ * @returns {Icon} - an emoji passed into a string depending on which file
+ * type was found (FILE, CONFIG_FILE, or FOLDER)
+*/
+
+export enum Icon {
+  /* Added one space after emoji so that it doesn't stick together
+  with the next phrase or delete a character */
+  FOLDER = '📂 ',
```
Comment (ralph-dev) [inline src/utils/getFileIconFromFileType.ts:14]: Remove the string space and have a separate function for string formatting

Comment (Manpreet-Bhatti) [reply src/utils/getFileIconFromFileType.ts]: Created a helper function to format emoji so that space is added after it

**Pull request merged**